content/roadmap/roadmap.md

```diff
@@ -5,11 +5,14 @@ weight: 81
 lead: ""
 draft: false
 images: []
+url: /roadmap/roadmap
 toc: true
 sidebar:
   collapsed: true
 ---
 
+## Our Roadmap
+
 You can checkout the [Project Roadmap on GitHub](https://github.com/orgs/open-component-model/projects/10/views/5) which is based on issues and PRs from the [OCM project repository](https://github.com/open-component-model/ocm-project).
 
-![roadmap](/images/roadmap_Q2-2024.png)
+![roadmap](/images/roadmap_Q2-2025.png)
```

Comment:
See my comment that we dont really need this roadmap picture here

Reply:
See my comment on your first comment :-) about nearly empty page vs. one with pic. One other option also would be to remove the Roadmap section completely. WDYT?

Reply:
I think we could just link to the roadmap in our general header or on another page, instead of offering a completely new one. WDYT?

Reply:
ok, I changed the setup from an own page with some text and the pic and just placed the link to https://github.com/orgs/open-component-model/projects/10/views/5 behind the top level "Roadmap". Fits for you?

Reply:
For me this is fine, but the loading time was unexpected, maybe you wanna denote with an icon that this leads to github?

Reply:
no really :-). I don't experience a notable loading time on GitHub and if, this is because of GitHub, not the website.  When clicking on "Roadmap" I'm immediately rerouted to Github, so anybody clicking on the link should notice that he leaves the site. But the main reason is that a) an additional Icon for GitHub does not fit to the current design (as we have icons for that already on the right side for social interaction) and b) it's not that easy to get this into the current (and future) theme. All such things would need to go into multiple sections (facials, menus.toml, params.toml) which is exactly what we wanted to avoid: The whole idea of keeping the page as maintainable as possible, is to choose a theme that works out of the box with as few adoptions in the html sections.